suite-native/accounts/src/components/QRCode.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
We should probably create package for this or maybe move this to atoms. (I will check it later today)

Reply:
Please rename package `qr-code-scanner` which we already have to just `qr-code` and move it there.

Reply:
https://github.com/trezor/trezor-suite/pull/7178/commits/35bd1d327d3c7cead98f2ffdb95111f887c7ee68